.github/workflows/images-vulnerability-scanning.yaml

```diff
@@ -0,0 +1,58 @@
+name: Images vulnerability scanning
+permissions:
+  contents: read
+on:
+  workflow_call:
+    inputs:
+      images_file:
+        required: true
+        type: string
+
+jobs:
+  discover-images:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - name: Read JSON file
+        id: images-json
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "images_json<<$EOF" >> $GITHUB_OUTPUT
+          cat ${{ inputs.images_file }} >> $GITHUB_OUTPUT
+          echo "$EOF" >> $GITHUB_OUTPUT
+      - name: Showing output variable
+        run: echo ${{fromJson(steps.images-json.outputs.images_json)}}
+    outputs:
+      images-json: ${{steps.images-json.outputs.images_json}}
+  report-analysis:
+    runs-on: ubuntu-20.04
+    needs:
+      - discover-images
+    strategy:
+      max-parallel: 3
+      fail-fast: false
+      matrix:
+        images: ${{fromJson(needs.discover-images.outputs.images-json).images}}
+    name: ${{ matrix.images }}
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - name: Printing Image Registry
+        id: image-registry
+        run: echo "image_registry=${{fromJson(needs.discover-images.outputs.images-json).image_registry}}" >>  "$GITHUB_ENV"
+      - name: Printing Image Tag
+        id: image-tag
+        run: echo "image_tag=${{fromJson(needs.discover-images.outputs.images-json).tag}}" >>  "$GITHUB_ENV"
+      - name: Printing Image Path
+        run: echo "image_path=${{env.image_registry}}/${{matrix.images}}:${{env.image_tag}}" >> "$GITHUB_ENV"
+      - name: Running vulnerability scanner
+        uses: anchore/scan-action@v3
+        id: vulnerability-scanning
+        with:
+          image: ${{env.image_path}}
+          fail-build: false
+          output-format: json
+          only-fixed: true
```

Comment:
Will it still be possible to notice issues that aren't fixed by upstream? In many cases it might be possible to provide a mitigation

Reply:
I'm not going to change that in this PR. We can discuss our strategy in relation to non-fixed upstream vulnerabilities later.